tutorials/v7/draw_rh1.cxx

```diff
@@ -45,8 +45,13 @@ void draw_rh1() {
 
    // Create a canvas to be displayed.
    auto canvas = RCanvas::Create("Canvas Title");
-   canvas->Draw(pHist)->Line().SetColor(RColor::kRed);
-   canvas->Draw(pHist2)->Line().SetColor(RColor::kBlue);
+   auto draw1 = canvas->Draw(pHist);
```

Comment:
Should we rename `draw` to `opts`?

Reply:
With such syntax `Line().SetColor(RColor::kRed).SetWidth(2)` one could skip variable completely.